Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -123,6 +126,26 @@ void WiimoteControllersWidget::CreateLayout()
   // Passthrough BT
   m_wiimote_layout->addWidget(m_wiimote_passthrough, m_wiimote_layout->rowCount(), 0, 1, -1);
 
+  int adapter_row = m_wiimote_layout->rowCount();
+  m_wiimote_layout->addWidget(m_bluetooth_adapters_label, adapter_row, 1, 1, 1);
+  m_wiimote_layout->addWidget(m_bluetooth_adapters, adapter_row, 2, 1, 2);
+  const int selected_vid = Config::Get(Config::MAIN_BLUETOOTH_PASSTHROUGH_VID);
+  const int selected_pid = Config::Get(Config::MAIN_BLUETOOTH_PASSTHROUGH_PID);
+  for (auto device : IOS::HLE::BluetoothRealDevice::ListDevices())
```

Comment:
I think you should add an `Automatic` entry at the top with VID/PID -1:-1 so users can still configure the default state of not having an adapter explicitly chosen for them. Might need some refactoring to handle the negative value though.